server/ntb/publish/ntb_nitf.py

```diff
@@ -148,6 +225,16 @@ def repl_embedded(match):
         # and etree from stdlib doesn't have a proper HTML parser
         soup = BeautifulSoup(html, 'html.parser')
         html_elts = ET.fromstring('<div>{}</div>'.format(soup.decode(formatter='xml')))
+        # <p class="lead" lede="true"> is used by NTB for abstract
```

Comment:
the lede="true" is used on the first <p> and they want the abstract there (with class="lead").

We can also get <p class="lead"> on the first paragraph from client (it may come from the ingest, not sure if it's the client or the ingest), so I need to remove to avoid 2 "lead". But there will still be a <p class="lead" lede="true"> as first paragraph, just it will come from the abstract and not the body_html.